Neos.Media/Classes/Domain/Service/ThumbnailService.php

```diff
@@ -78,31 +75,17 @@ class ThumbnailService
     protected $thumbnailCache = [];
 
     /**
+     * @Flow\Inject
      * @var LoggerInterface
      */
     private $logger;
```

Comment:
Either this
```suggestion
    protected $logger;
```
or keep the method injection…